pkgs/development/tools/google-java-format/default.nix

```diff
@@ -17,11 +17,17 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out/{bin,share/google-java-format}
-    install -D ${src} $out/share/google-java-format/google-java-format.jar
-
-    makeWrapper ${jre}/bin/java $out/bin/google-java-format \
-      --add-flags "-jar $out/share/google-java-format/google-java-format.jar"
+    mkdir -p $out/{bin,${pname}/share}
```

Comment:
This should be `share/${pname}` (but also probably redundant because of `install -D` makes it work)

(I am starting to think I am too annoying with too many comments)

Reply:
I just made that revision too quickly, thanks for catching it. Hopefully it's good to go now.